bin/main_pysam.py

```diff
@@ -28,9 +28,14 @@ def main():
     header_written = False
     strand_counter = Counter()
     code_counter = Counter()
+   
     if args.hm_sumstats:
         out_handle = open_gzip(args.hm_sumstats, "wb")
         out_header = SumStatsTable(sumstats_file=args.sumstats)._set_header_order()
+        tag_neg_log_10_p_value=False
+        if "neg_log_10_p_value" in out_header:
```

Comment:
`"neg_log_10_p_value"` could be a constant.

Reply:
Thanks for your comments! Yes, you are right! Let me mark it in the next release! 
- add "neg_log_10_p_value" as a constant in the bin/common_constants.py